validate_tests.py

```
@@ -0,0 +1,61 @@
+    # Count test functions
+    total_tests = 0
+    for test_file in test_files:
+        try:
+            with open(test_file, 'r') as f:
+                content = f.read()
+            tree = ast.parse(content)
+            
+            for node in ast.walk(tree):
+                if isinstance(node, ast.FunctionDef) and node.name.startswith('test_'):
+                    total_tests += 1
+        except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)